custom_components/tuya_local/devices/sime_ecomaxi_vb200_heatpump.yaml

```diff
@@ -0,0 +1,180 @@
+      - id: 4
+        name: temperature
+        type: integer
+        unit: C
+        range:
+          min: 35
+          max: 65
+      - id: 16
+        name: current_temperature
+        type: integer
+        unit: C
+  - entity: switch
+    name: Power
+    icon: "mdi:power"
+    dps:
+      - id: 1
+        type: boolean
+        name: switch
+  - entity: switch
+    name: Anti-legionella
+    icon: "mdi:bacteria"
+    category: config
+    dps:
+      - id: 9
+        type: boolean
+        name: switch
+  - entity: switch
+    name: Electric anode
+    icon: "mdi:heat-wave"
+    category: config
+    dps:
+      - id: 101
+        type: boolean
+        name: switch
+  - entity: sensor
+    name: Evaporator temperature
+    class: temperature
+    category: diagnostic
+    dps:
+      - id: 23
+        type: integer
+        name: sensor
+        unit: C
+  - entity: sensor
+    name: Air outlet temperature
+    class: temperature
+    category: diagnostic
+    dps:
+      - id: 24
+        type: integer
+        name: sensor
+        unit: C
+  - entity: sensor
+    name: Ambient temperature
+    class: temperature
+    category: diagnostic
+    dps:
+      - id: 26
+        type: integer
+        name: sensor
+        unit: C
+  - entity: sensor
+    name: Solar collector temperature
+    class: temperature
+    category: diagnostic
+    dps:
+      - id: 102
+        type: integer
+        name: sensor
+        unit: C
+  - entity: sensor
+    name: EEV valve opening
+    icon: "mdi:valve"
+    category: diagnostic
+    dps:
+      - id: 113
+        type: integer
+        name: sensor
+  - entity: sensor
+    name: Main board temperature
+    icon: "mdi:chip"
+    category: diagnostic
+    dps:
+      - id: 111
+        type: integer
+        name: sensor
+        unit: C
+  - entity: sensor
+    name: Display board temperature
+    icon: "mdi:monitor"
+    category: diagnostic
+    dps:
+      - id: 112
+        type: integer
+        name: sensor
+        unit: C
+  - entity: binary_sensor
+    name: Water pump
+    icon: "mdi:pump"
+    class: running
+    category: diagnostic
+    dps:
+      - id: 28
+        type: boolean
+        name: sensor
+  - entity: binary_sensor
+    name: Compressor
+    icon: "mdi:engine"
+    class: running
+    category: diagnostic
+    dps:
+      - id: 29
+        type: boolean
+        name: sensor
+  - entity: binary_sensor
+    name: Fan
+    icon: "mdi:fan"
+    class: running
+    category: diagnostic
+    dps:
+      - id: 27
+        type: boolean
+        name: sensor
+  - entity: binary_sensor
+    name: Defrost
+    icon: "mdi:snowflake-melt"
+    class: running
+    category: diagnostic
+    dps:
+      - id: 32
+        type: boolean
+        name: sensor
+  - entity: binary_sensor
+    name: Electric heater
+    icon: "mdi:lightning-bolt"
+    class: heat
+    category: diagnostic
+    dps:
+      - id: 33
+        type: boolean
+        name: sensor
+  - entity: sensor
```

Comment:
I think it is better to follow the standard pattern used in other configs of a binary_sensor with `class: problem`, the sensor mapped to false for 0, true for any other value, and a fault_code attribute with the raw value. Generally this type of field is a bitfield (bitmap in Tuya docs) rather than an integer, but the difference only matters when you start mapping non-zero values.